examples/asa/scripts/transfer/gold-delegated-lsig.js

```diff
@@ -18,7 +18,7 @@ async function run (runtimeEnv, deployer) {
   let txnParam = {
     type: types.TransactionType.TransferAsset,
     sign: types.SignType.LogicSignature,
-    fromAccount: goldOwner,
+    fromAccountAddr: goldOwner.addr,
```

Comment:
Why sometimes we have `.address` and other times `.addr`?

Reply:
In runtime's *AccountStore*, we have acc.account (which is the account with `addr`, `sk`) & acc.address as well. 

`.address` is being used in runtime tests, and otherwise with deployer we are using `account.addr` (eg. goldOwner.addr)